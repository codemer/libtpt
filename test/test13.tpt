This test generates a macro.
@macro (test) {
This is the macro content.
}
@test()\
And that was the macro.
