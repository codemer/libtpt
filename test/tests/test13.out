This test generates a macro.
This is the macro content.
And that was the macro.
